api/src/test/scala/hmda/api/http/institutions/SubmissionPathsSpec.scala

```diff
@@ -102,8 +102,8 @@ class SubmissionPathsSpec extends InstitutionHttpApiSpec {
     val expectedEdits =
       EditResults(
         List(
-          EditResult("V285", List(LarEditResult(LarId("loan2")), LarEditResult(LarId("loan3")))),
-          EditResult("V280", List(LarEditResult(LarId("loan1"))))
+          EditResult("V285", ts = false, List(LarEditResult(LarId("loan2")), LarEditResult(LarId("loan3")))),
```

Comment:
This endpoint also needs a test to demonstrate that TS errors are published correctly